### PR TITLE
add boilerplate and a paragraph

### PR DESCRIPTION
**Pull request by yusufalp**

### CODE REVIEW

**Review by yusufalp — COMMENTED** [review]
Can you add another paragraph please?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <p>We are working on making a PR</p>
```
Comment (yusufalp) [inline index.html:9]: Add a new paragraph about the name of the person here.

Comment (yusufalp) [reply index.html:9]: Done!

**Review by Uni-syeda — COMMENTED** [review]
Excellent work!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    <p>Yusuf Alp</p>
```
Comment (Npcodes1) [inline index.html:10]: Can you add a date?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
```
Comment (Angelliz96) [inline index.html:8]: insert an image here, please

**Review by jasminearnold — COMMENTED** [review]
great job!

File: index.html (hunk 2 above)
Comment (yusufalp) [reply index.html:10]: Done!

File: index.html (hunk 3 above)
Comment (yusufalp) [reply index.html:8]: That's too much.

**Comment by kaelisaac** [comment]
maybe add a heading?

**Review by jasminearnold — APPROVED** [review]

**Review by Npcodes1 — APPROVED** [review]

**Review by Angelliz96 — APPROVED** [review]

**Review by jasmineeuy — APPROVED** [review]
Great Changes!

**Review by leodjones — APPROVED** [review]

**Review by Uni-syeda — APPROVED** [review]

**Pull request merged**